packages/react-geo/jest.config.js

```diff
@@ -3,19 +3,20 @@ module.exports = {
   collectCoverageFrom: ['<rootDir>/src/**/*.(ts|tsx)'],
   // ignore coverage for top level "export", version and style files
   coveragePathIgnorePatterns: ['<rootDir>/src/(index|styles|version).(ts|tsx)'],
-  coverageThreshold: {
-    global: {
-      branches: 80,
-      functions: 80,
-      lines: 80,
-      statements: 80,
-    },
-  },
+  // coverageThreshold: {
+  //   global: {
+  //     branches: 80,
+  //     functions: 80,
+  //     lines: 80,
+  //     statements: 80,
+  //   },
+  // },
```

Comment:
Instead of commenting out why not add the thresholds to what we have now so we at least have a baseline?

Reply:
Sure, will add that for now.
Also added a pending item in PR description that lists current coverage against expected coverage.